rllib/examples/learners/mog_learner.py

```diff
@@ -0,0 +1,134 @@
+"""Example of how to write a custom loss function, custom rl_module and custom config.
+
+This example shows:
+    - how to subclass an existing (torch) Learner and override its `compute_loss_for_module()` method
+    - how you can add your own loss to a custom (or non-custom) rl_module using compute_loss_for_module
+    - how to add add custom pipeline components to a custom config class using build_learner_connector
+    - how to plug in a custom Learner, rl_module, its loss, and use a custom Config to train with
+
```

Comment:
Sorry for nagging here: could we describe a bit more what this example uses b/c this is the really interesting part of it: Mixture of Gaussians and why it is so interesting? For a great example you could take a look athe ICR: https://github.com/simonsays1980/ray/blob/348589b4942768863d14968900b4d9e7d1340e9d/rllib/examples/curiosity/intrinsic_curiosity_model_based_curiosity.py#L1